components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -7935,6 +7935,7 @@ def PrototypeList(config):
             CGGeneric(AUTOGENERATED_WARNING_COMMENT),
             CGGeneric(f"pub const PROTO_OR_IFACE_LENGTH: usize = {len(protos) + len(constructors)};\n"),
             CGGeneric(f"pub const MAX_PROTO_CHAIN_LENGTH: usize = {config.maxProtoChainLength};\n\n"),
+            CGGeneric("#[allow(clippy::enum_variant_names)]\n"), 
```

Comment:
```suggestion
            CGGeneric("#[allow(clippy::enum_variant_names)]"), 
```

The explicit newline (`\n`)  is not needed here. With this explicit newline, the generated `PrototypeList.rs` will have a blank line between this `allow` and the rest of the enum. Technically the extra newline doesn't change the sematics, but it does make the generated code look "wrong".

Reply:
Understood! I shall fix this